lib/railway_ipc/consumer/consumer_response_handlers.rb

```diff
@@ -0,0 +1,14 @@
+require 'railway_ipc/handler_store'
```

Comment:
Pattern followed to store handlers accessed by both the class and instance of a Consumer. There's a corresponding class for Client and Server.

As noted previously, all classes inheriting from Consumer will share this singleton. It may be a problem.